src/api/apiClient.ts

```diff
@@ -209,7 +209,7 @@ class ApiClient {
     method: string,
     endpoint: string,
     options: RequestOptions = {},
-    config: RequestConfiguration & { cache?: CacheConfiguration } = {},
+    config: RequestConfiguration & { cache?: CacheConfiguration; asRawResponse?: boolean } = {},
```

Comment:
Not sure I'm happy with all these extensions to the API Client, maybe there are better solutions.